tools/walletextension/wallet_extension.go

```diff
@@ -44,7 +44,7 @@ func New(
 		hostAddr:           hostAddr,
 		userAccountManager: userAccountManager,
 		unsignedVKs:        map[gethcommon.Address]*rpc.ViewingKey{},
-		storage:            storage,
+		Storage:            storage,
```

Comment:
Storage should be a private prop. Only the WE should access it.

Reply:
storate is private again, moved all the code accessing it to WE